skbio/stats/composition.py

```diff
@@ -1146,81 +1147,57 @@ def tree_basis(tree):
     return basis, nodes
 
 
-def _holm_bonferroni(p):
-    """Perform Holm-Bonferroni correction.
-
-    Perform Holm-Bonferroni correction ([1]_) for *p*-values to account for
-    multiple comparisons.
+def _calc_p_adjust(name, p):
+    """
+    Calculate the p-value adjustment for a given method.
 
     Parameters
-    ----------
-    p : ndarray of shape (n_tests,)
-        Original *p*-values.
+    -------
```

Comment:
There are some issues with the format of the docstring. Despite that it won't be linted because it belongs to a helper function (starting with `_`), it is still a good habit to write docstring in a nice way.

Reply:
Oh yes, I may have overlooked some aspects of the formatting. I have made the necessary changes and ensured they align with the rest.